hello-world/src/test/java/org/acme/schooltimetabling/solver/TimetableFastAssertTest.java

```diff
@@ -0,0 +1,41 @@
+package org.acme.schooltimetabling.solver;
+
+import static org.acme.schooltimetabling.TimetableApp.generateDemoData;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+
+import ai.timefold.solver.core.api.solver.Solver;
+import ai.timefold.solver.core.api.solver.SolverFactory;
+import ai.timefold.solver.core.config.solver.EnvironmentMode;
+import ai.timefold.solver.core.config.solver.SolverConfig;
+
+import org.acme.schooltimetabling.TimetableApp;
+import org.acme.schooltimetabling.domain.Lesson;
+import org.acme.schooltimetabling.domain.Timetable;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("slowly")
+class TimetableFastAssertTest {
```

Comment:
So we are no longer doing `FULL_ASSERT`? I thought we'd be doing both.

Reply:
Should we have tests for both environments?

Reply:
I was expecting that.